src/rest-server/src/controllers/v2/user.js

```diff
@@ -270,9 +256,6 @@ const addGroupIntoUserGrouplist = async (req, res, next) => {
     const username = req.params.username;
     const groupname = req.body.groupname;
     let userInfo = await userModel.getUser(username);
-    if (userInfo.grouplist.includes(authConfig.groupConfig.adminGroup.groupname)) {
```

Comment:
Allow the modify for admin user?

Reply:
There are two questions:
1. Whether allow to update user permission by update grouplist.
Admin is a group attr, not a user attr, so allow to update user admin by updating his groups make sense. If we stick origin experience, I'm fine, but the logic is disallowing to add/remove *admin group* into/from user grouplist.
2. Whether allow to update admin grouplist if no admin role change.
I think it should be. E.g If we support different group storage preference(not permission), we should also allow to customize such preference for admin, the solution is add admin into a specific group configured his preference.

Reply:
sync with @scarlett2018